server/web_ui/admin/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Is there a reason to split this up between the packages? Could we just have a single "build all the wasms?"
 Or just a way to symlink them all to a common script?

Reply:
Sorted 😄